package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@observation.org/react-native-components",
+  "version": "0.0.35",
+  "main": "lib/commonjs/index.js",
+  "repository": "git@github.com:observation/react-native-components.git",
+  "author": "Jonathan <jonathan@waarneming.nl>",
+  "license": "MIT",
+  "devDependencies": {
+    "@react-native-community/eslint-config": "^3.2.0",
+    "@testing-library/react-native": "^12.1.2",
+    "@tsconfig/react-native": "^3.0.2",
+    "@types/color": "^3.0.3",
+    "@types/jest": "^29.5.2",
+    "@types/react": "^18.2.14",
+    "@types/react-native": "^0.72.2",
+    "eslint": "^8.19.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-observation": "https://github.com/observation/eslint-rules.git",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-react-native": "^4.0.0",
+    "jest": "^29.5.0",
+    "prettier": "^2.8.8",
+    "react": "18.2.0",
+    "react-native": "^0.71.0",
+    "react-native-builder-bob": "^0.20.4",
+    "react-test-renderer": "^18.2.0",
+    "ts-jest": "^29.1.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4"
+  },
+  "react-native": "src/index.ts",
+  "module": "lib/module/index.js",
+  "types": "lib/typescript/index.d.ts",
+  "scripts": {
+    "test": "jest",
+    "prepack": "bob build && rm -rf lib/typescript/components/__tests__"
+  },
+  "files": [
+    "lib",
+    "src"
+  ],
+  "eslintIgnore": [
+    "node_modules/",
+    "lib/"
+  ],
+  "peerDependencies": {
+    "react": "18.2.0",
+    "react-native": "^0.71.0"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.3.0",
+    "@fortawesome/pro-light-svg-icons": "^6.3.0",
+    "@fortawesome/pro-solid-svg-icons": "^6.3.0",
+    "@fortawesome/react-native-fontawesome": "^0.3.0",
+    "@react-native-community/blur": "^4.2.0",
+    "@react-native-firebase/crashlytics": "^17.0.0",
```

Comment:
Deze dependency zou ik niet verwachten, hoort aan applicatiekant thuis. Wordt gebruikt in de logger. Zouden we de configuratie van Logger niet kunnen inserten?